ui/mainWindow.py

```diff
@@ -22,11 +24,11 @@ def setupUi(self, MainWindow):
         self.controlLayout = QtWidgets.QVBoxLayout()
         self.controlLayout.setSizeConstraint(QtWidgets.QLayout.SetMinimumSize)
         self.controlLayout.setObjectName("controlLayout")
-        self.label = QtWidgets.QLabel(self.centralwidget)
-        self.label.setMaximumSize(QtCore.QSize(16777215, 29))
-        self.label.setOpenExternalLinks(True)
-        self.label.setObjectName("label")
-        self.controlLayout.addWidget(self.label)
+        #self.label = QtWidgets.QLabel(self.centralwidget)
+        #self.label.setMaximumSize(QtCore.QSize(16777215, 29))
+        #self.label.setOpenExternalLinks(True)
+        #self.label.setObjectName("label")
+        #self.controlLayout.addWidget(self.label)
```

Comment:
This file cannot be edited manually. It is derived from ui files

Please make changes to the ui file using the ut development tools and run the `build.sh ui`

```
# Form implementation generated from reading ui file 'ui/mainWindow.ui'
#
# Created by: PyQt5 UI code generator 5.15.1
#
# WARNING: Any manual changes made to this file will be lost when pyuic5 is
# run again.  Do not edit this file unless you know what you are doing.
```

Reply:
And why need to delete it

Reply:
Wait, so i need to redo the .py changes onto the .ui files?

Reply:
They are some changes that i don't know how to redo in QT Designer?

Reply:
The workflow for working with files is such that first their basis is created in the development tool QT Designer
Then the python code generation script pyuic5 is launched

All changes that cannot be made in the designer must be made by changing class attributes or added via inheritance. Without changing the generated base class, because it is a product of generation from ui files